### DISCUSSION TITLE
yiwen-blog-website/blog/2024_0607

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#264] [Announcements]
# yiwen-blog-website/blog/2024_0607

马上25岁了，7年前走出家门来到北京，再到现在因为机缘巧合来到上海。各类事情萦绕身旁，还好，都走过来了。

https://hao-yiwen.github.io/yiwen-blog-website/blog/2024_0607

**Comment by Hao-yiwen** [comment]
ai > rn = flutter > android > ios
